examples/copy/Dockerfile

```diff
@@ -269,37 +271,39 @@ RUN test $(stat -c '%A' dir19/dir19a2 | cut -c8-) != 'rwx' \
 
 ## Destination: Symlink, 2nd level.
 
-# Note: This behavior is DIFFERENT from the symlink at 1st level tests above
-# (recall we are trying to be bug-compatible with Docker).
-
-# Set up destination.
-RUN mkdir dir20 \
- && echo new > dir20/filex \
- && mkdir dir20/dirx \
- && for i in $(seq 4); do \
-           echo file$i > dir20/file$i \
-        && ln -s file$i dir20/s_file$i \
-        && mkdir dir20/dir$i \
-        && echo dir$i/file_ > dir20/dir$i/file_ \
-        && ln -s dir$i dir20/s_dir$i; \
-    done \
- && ls -lR dir20
-
-# Copy in the new directory tree. In all of these cases, the source simply
-# overwrites the destination; symlinks are not followed.
+# NOTE: Not currently tested (see #1707).
```

Comment:
You might want to add some extra context for this comment chunk, particularly the commented out Dockerfile instruction that's still included.